lib/block-supports/elements.php

```diff
@@ -105,17 +105,13 @@ function gutenberg_render_elements_support_styles( $pre_render, $block ) {
 	$link_block_styles = isset( $element_block_styles['link'] ) ? $element_block_styles['link'] : null;
 
 	if ( $link_block_styles ) {
-		$styles = gutenberg_style_engine_generate(
+		gutenberg_style_engine_enqueue_styles(
```

Comment:
The reason this draft PR is hard to follow is because this function is missing and not defined anywhere 😅

Reply:
Ha. You got me. 

This stuff should be better documented. I'll address that in the README.md file. 

So it's _actually_ calling a renamed `wp_style_engine_enqueue_styles`, found [here](https://github.com/WordPress/gutenberg/pull/41424/files#diff-69616e5e4ff1a06362db0b7165c330b00614d4ea0d3a28c80003c0860ab8b93fR580).

The `gutenberg_` prefix is added during the plugin build step via some [webpack voodoo](https://github.com/WordPress/gutenberg/blob/trunk/tools/webpack/packages.js#L61). 

So internally, the plugin fires `gutenberg_style_engine_enqueue_styles` while in WordPress core it'll eventually fire `wp_style_engine_enqueue_styles`. It's a migration pattern that's used throughout the plugin.